lavalab-gen.py

```diff
@@ -92,6 +92,10 @@ def main(args):
                 baud = b["uart"].get("baud", baud_default)
                 idvendor = b["uart"]["idvendor"]
                 idproduct = b["uart"]["idproduct"]
+                # Prevent that convertion of the hex value
+                if type(idproduct) == int or type(idvendor) == int:
+                    print("Please encapsulate IDs for %s with quotes" % board_name)
+                    sys.exit(1)
```

Comment:
Instead, read the values as hex (which will valditate they are hex) and then print them in udev rules as hex with `%04x`

Reply:
print as hex is done, but we cannot read as hex, only as int so we cannot know if the value was with prefixed wit 0x or not